src/tqec/plaquette/__init__.py

```diff
@@ -0,0 +1,25 @@
+from .plaquette import (
+    Plaquette,
+    SquarePlaquette,
+    RoundedPlaquette,
+    PlaquetteList,
+)
+
+from .qubit import PlaquetteQubit
+
+from .schedule import (
+    ScheduleException,
+    ScheduledCircuit,
+    ScheduledCircuits,
```

Comment:
The `ScheduledCircuits` class is only an implementation detail and does not appear in the public API, I am in favour of removing it from the top-level imports.
```suggestion
```